File: app/src/main/java/com/kota/TelnetUI/TextView/TelnetTextViewLarge.java
package com.kota.TelnetUI.TextView;

import android.content.Context;
import android.util.AttributeSet;

public class TelnetTextViewLarge extends TelnetTextView {
    public TelnetTextViewLarge(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
        reloadTextSize();
    }

    public TelnetTextViewLarge(Context context, AttributeSet attrs) {
        super(context, attrs);
        reloadTextSize();
    }

    public TelnetTextViewLarge(Context context) {
        super(context);
        reloadTextSize();
    }

    protected void reloadTextSize() {
        setTextModelSize(TEXT_VIEW_SIZE_LARGE);
    }
}

File: app/src/main/java/com/kota/TelnetUI/TextView/TelnetTextViewLarge.kt
package com.kota.TelnetUI.TextView

import android.content.Context
import android.util.AttributeSet

class TelnetTextViewLarge : TelnetTextView {
    constructor(context: Context?, attrs: AttributeSet?, defStyle: Int) : super(
        context,
        attrs,
        defStyle
    ) {
        reloadTextSize()
    }

    constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs) {
        reloadTextSize()
    }

    constructor(context: Context?) : super(context) {
        reloadTextSize()
    }

    override fun reloadTextSize() {
        setTextModelSize(TelnetTextView.Companion.TEXT_VIEW_SIZE_LARGE)
    }
}
